fission/src/ui/panels/configuring/scoring/ZoneConfigPanel.tsx

```diff
@@ -1,84 +1,202 @@
-            <Checkbox label="Destroy Gamepiece" defaultState={false} onClick={setDestroy} />
-            <Checkbox label="Persistent Points" defaultState={false} onClick={setPersistent} />
-            <Slider
-                label="X Scale"
-                min={0}
-                max={10}
-                defaultValue={1}
-                format={{ maximumFractionDigits: 2 }}
-                onChange={(v: number) =>
-                    setScale(s => {
-                        s[0] = v
-                        return s
-                    })
-                }
-            />
-            <Slider
-                label="Y Scale"
-                min={0}
-                max={10}
-                defaultValue={1}
-                format={{ maximumFractionDigits: 2 }}
-                onChange={(v: number) =>
-                    setScale(s => {
-                        s[1] = v
-                        return s
-                    })
-                }
-            />
-            <Slider
-                label="Z Scale"
-                min={0}
-                max={10}
-                defaultValue={1}
-                format={{ maximumFractionDigits: 2 }}
-                onChange={(v: number) =>
-                    setScale(s => {
-                        s[2] = v
-                        return s
-                    })
-                }
-            />
+        <Panel
+            name="Scoring Zone Config"
+            panelId={panelId}
+            openLocation={openLocation}
+            sidePadding={sidePadding}
+            onAccept={() => {
+                saveSettings()
+                if (transformGizmoRef.current) transformGizmoRef.current.RemoveGizmos()
+                openPanel("scoring-zones")
+            }}
+            onCancel={() => {
+                openPanel("scoring-zones")
+                if (transformGizmoRef.current) transformGizmoRef.current.RemoveGizmos()
+            }}
+        >
+            <div className="flex flex-col gap-2 bg-background-secondary rounded-md p-2">
+                {/** Set the zone name */}
+                <Input
+                    label="Name"
+                    placeholder="Enter zone name"
+                    defaultValue={SelectedZone.zone.name}
+                    onInput={setName}
+                />
+
+                {/** Set the alliance color */}
+                <Button
+                    value={`${alliance[0].toUpperCase() + alliance.substring(1)} Alliance`}
+                    onClick={() => setAlliance(alliance == "blue" ? "red" : "blue")}
+                    colorOverrideClass={`bg-match-${alliance}-alliance`}
+                />
+
+                {/** Select a parent node */}
+                <SelectButton
+                    placeholder="Select parent node"
+                    onSelect={(body: Jolt.Body) => trySetSelectedNode(body.GetID())}
+                />
```

Comment:
Currently, you can't select the root node of the field, and in addition, it doesn't indicate which node you've selected. See intake or ejector for details on how to do that.

Reply:
@HunterBarclay is there a reason to select the root node of a field because it won't ever move?